pivotal-import/README.md

```diff
@@ -44,14 +47,14 @@ You can run `make clean` if you want to start over, but be aware this will delet
 
 The following are known limitations:
 
-- **Limited story reviews:** Shortcut does not have a feature equivalent to Pivotal story reviews, so they are imported as follows:
+- **Story reviews:** Shortcut does not have a feature equivalent to Pivotal story reviews, so they are imported as follows:
   - Pivotal story reviewers are imported as Shortcut story followers on the stories they were assigned for review. Shortcut story followers receive updates in their Shortcut Activity Feed for all story updates.
   - Imported stories that had Pivotal reviews have an additional comment with a table that lists all of the story reviews from Pivotal (reviewer, review type, and review status).
   - Imported stories that had Pivotal reviews have a label in Shortcut of `pivotal-had-review`.
+- **File attachments:** Files directly uploaded to Pivotal and included in its export are detected and uploaded to the corresponding imported Shortcut stories. Other kinds of attachments (e.g., Google Drive) are not supported.
```

Comment:
Recommend mentioning that files _associated with imported stories_ are detected and uploaded. While it is supposed to be equivalent it's more aligned with the mechanism we've chosen.